_includes/header--help.html

```diff
@@ -97,7 +97,7 @@
         {% include language_picker.html dropdown=true id="dropdown" %}
       </div>
 
-      {% include search.html id="header-nav" %}
+      {% include search.html id="header-nav" collection="8801" %}
```

Comment:
Is my understanding correct that 8801 and 8803 are given values from search.gov for the brochure and partners site respectively?

Reply:
Yep, if you sign in to Search.gov, you can see them from URLs for "Edit" links at this page: 

https://search.usa.gov/sites/8022/collections

"For partners": https://search.usa.gov/sites/8022/collections/8803/edit
"Login.gov account help": https://search.usa.gov/sites/8022/collections/8801/edit